c/include/ml-api-service.h

```diff
@@ -38,27 +38,63 @@ extern "C" {
 /**
  * @brief Sets the pipeline description with a given name.
  * @since_tizen 7.0
+ * @remarks If the name already exists, the pipeline description is overwritten.
  * @param[in] name Unique name to retrieve the associated pipeline description.
- * @param[in] pipeline_desc The pipeline description to be stored
+ * @param[in] pipeline_desc The pipeline description to be stored.
  * @return @c 0 on success. Otherwise a negative error value.
- * @note If the name already exists, the pipeline description is overwritten.
  * @retval #ML_ERROR_NONE Successful.
  * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
  * @retval #ML_ERROR_INVALID_PARAMETER Fail. The parameter is invalid.
- * @retval #ML_ERROR_IO_ERROR The operation of DB or filesystem is failed.
+ * @retval #ML_ERROR_IO_ERROR The operation of DB or filesystem has failed.
+ *
+ * Here is an example of the usage:
+ * @code
+ * const gchar my_pipeline[] = "videotestsrc is-live=true ! videoconvert ! tensor_converter ! tensor_sink async=false";
+ * gchar *description;
+ * int status;
+ * ml_pipeline_h handle;
+ *
+ * // Set pipeline description.
+ * status = ml_service_set_pipeline ("my_pipeline", my_pipeline);
+ * if (status != ML_ERROR_NONE) {
+ *   // handle error case
+ *   goto error;
+ * }
+ *
+ * // Example to construct a pipeline with stored pipeline description.
+ * // Users may register intelligence pipelines for other processes and fetch such registered pipelines.
+ * // For example, a developer adds a pipeline which includes preprocessing and invoking a neural network model,
+ * // then an application can fetch and construct this for intelligence service.
+ * status = ml_service_get_pipeline ("my_pipeline", &pipeline);
+ * if (status != ML_ERROR_NONE) {
+ *   // handle error case
+ *   goto error;
+ * }
+ *
+ * status = ml_pipeline_construct (pipeline, NULL, NULL, &handle);
+ * if (status != ML_ERROR_NONE) {
+ *   // handle error case
+ *   goto error;
+ * }
+ *
+ * error:
+ * ml_pipeline_destroy (handle);
+ * g_free (pipeline);
+ * @endcode
```

Comment:
It's optional but how about adding `ml_service_delete_pipeline` for the example.